packages/editor/src/plugins/fill-in-the-blanks-exercise/components/blank-draggable-dummy-area.tsx

```diff
@@ -0,0 +1,58 @@
+import { faPlus } from '@fortawesome/free-solid-svg-icons'
+import { FaIcon } from '@serlo/frontend/src/components/fa-icon'
+
+import { BlankDraggableDummyAnswer } from './blank-draggable-dummy-answer'
+import type { FillInTheBlanksExerciseProps } from '..'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+
+interface BlankDraggableDummyAreaProps {
+  additionalDraggableAnswers: FillInTheBlanksExerciseProps['state']['additionalDraggableAnswers']
+}
+
+export function BlankDraggableDummyArea(props: BlankDraggableDummyAreaProps) {
+  const { additionalDraggableAnswers } = props
+
+  const blanksExerciseStrings = useEditorStrings().plugins.blanksExercise
+
+  const dummyDraggables = additionalDraggableAnswers.map(
```

Comment:
maybe `dummyValues` would be clearer here? I was expecting something more complex when reading `dummyDraggables`.